packages/core/src/parser/macros.ne

```diff
@@ -0,0 +1,13 @@
+sepBy1[ITEM, SEP] -> $ITEM (_ $SEP _ $ITEM):* (_ $SEP):? {%
+  ([first, rest]) => {
+    const restNodes = rest.map((ts: any[]) => ts[3]);
+    return concat(first, ...restNodes);
+  }
+%}
+
+sepBy[ITEM, SEP] -> null {% d => [] %} | $ITEM (_ $SEP _ $ITEM):* (_ $SEP):? {%
```

Comment:
Start each of the two cases in separate lines for readability?

Reply:
Also super great to see redundant code extracted! Somehow `@include` used to give me troubles. Maybe I never used it properly? Glad that it works now!

Reply:
> Start each of the two cases in separate lines for readability?

Done!